doc/man/8/ceph.rst

```diff
@@ -560,7 +560,8 @@ Usage::
 
 	ceph osd create {<uuid>} {<id>}
 
-Subcommand ``new`` reuses a previously destroyed OSD *id*. The new OSD will
+Subcommand ``new`` creates a new OSD. It can be used to create a previously
+destroyed OSD. It reuses a previously destroyed OSD *id*. The new OSD will
```

Comment:
It can be used to create a new OSD or to recreate a previously destroyed OSD with a specific *id*.